<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SortWindow</class>
 <widget class="QMainWindow" name="SortWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="realisationLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортування:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backButton_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="insertElement">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>250</width>
      <height>203</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Field name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Field Value</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="contextLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>440</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Предметна область:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="contentBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>435</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Сервери</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дата і час</string>
     </property>
    </item>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>20</width>
      <height>251</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="insertButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставити елемент</string>
    </property>
   </widget>
   <widget class="QLabel" name="functionalLlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Функціонал: сортування</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sortType">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>395</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Select sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Heap sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Merge sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Insert sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quick sort</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Count sort (int only!)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Radix sort(int only!)</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>641</width>
      <height>411</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data center</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Rack</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Company</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>510</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортувати за:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sortingParamether">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>505</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>IP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Data center</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rack</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Company</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="sortButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортувати!</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1052</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
